### DISCUSSION TITLE
Enable custom CSS in the CP.

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2970] [Feature Requests]
A native mechanism for adding custom CSS to the CP would go a long way to allowing people to test and distribute solutions to #1590. 

Once the general accessibility issues are addressed, the same custom CSS feature would allow for the distribution of CSS that addresses individual accessibility issues (example: CSS files tuned for specific types of color blindness).

**Comment by robsonsobral** [comment]
There's an [add-on](https://github.com/MaxLazar/cp-css-js) for that, but I think it's too powerful. It should be something configurable only by editing `config.php`.
